crates/l2/contracts/src/l1/OnChainProposer.sol

```diff
@@ -296,14 +297,17 @@ contract OnChainProposer is
         }
 
         lastVerifiedBatch = batchNumber;
-        // The first 2 bytes are the number of deposits.
-        uint16 deposits_amount = uint16(
+        // The first 2 bytes are the number of privileged transactions.
+        uint16 transactions_count = uint16(
```

Comment:
`transaction_count` is not accurate as it may be confused with total transaction